src/facade/reply_builder.h

```diff
@@ -57,6 +58,10 @@ class SinkReplyBuilder {
 
   virtual void SendProtocolError(std::string_view str) = 0;
 
+  // You normally should not call this - maps the status
+  // into the string that would be sent
+  static std::string_view StatusToMsg(OpStatus status);
```

Comment:
It's not your fault but those local fixes make the whole thing more confusing.
Let's do the following:
1. Move StatusToMsg to op_status.h as ` std::string_view StastusMessage(OpStatus status);`
2. let's make `SendError(ErrorReply error)` non virtual method in SinkReplyBuilder. 

I also changed how SendError is implemented for MC reply builder, I will send you a PR.

Reply:
> let's make SendError(ErrorReply error) non virtual method in SinkReplyBuilder.

We can't really do that because of  `void CapturingReplyBuilder::SendError(ErrorReply error)` actually overrides it with a different implementation.

Reply:
We overwrites the others as well, it just becomes a little less efficient because it can't move out of the string IIRC